### DISCUSSION TITLE
How to create a date range filter for a column?

### DISCUSSION THREAD

**Discussion by goldingdamien** [KevinVandy/material-react-table#262] [General]
I have a working date filter using below:

```
muiTableHeadCellFilterTextFieldProps: {
  type: "date",
}
```

The documents show ranges can be added with:

```
filterVariant: 'range',
filterFn: 'betweenInclusive',
```

However, I can't seem to get both working together. What is the correct way to implement this?
Regards.

**Comment by KevinVandy** [comment]
For advanced Filters like this, you will probably have to implement them yourself with the `Filter` component override. There is an example of this in the [Advanced Example](https://www.material-react-table.com/docs/examples/advanced) on the docs site

<img width="603" alt="image" src="https://user-images.githubusercontent.com/28243511/208129665-bb6da81b-0dd1-49c2-8897-c2773472cd78.png">

**Reply by goldingdamien** [reply]
Ok, thanks for your help.

**Comment by KevinVandy** [comment]
Date and Date-Range filters are now included in MRT v2 https://www.material-react-table.com/docs/examples/filter-variants
